src/components/Info/Form.js

```diff
@@ -142,13 +149,20 @@ export default ({ onSubmit, city }) => {
           />
         </label>
       )}
-
       <Button>
         I AM IN{' '}
         <span role="img" aria-label="Party">
           🎉
         </span>
       </Button>
+      //? this box displays the error text. A custom text is being diplayed but, can be changed to
```

Comment:
I am seeing this comment appear on the front end when I check out the deploy preview at https://deploy-preview-75--friendly-rosalind-7571fd.netlify.app/online-2021-june. Can you remove this comment?  ![IMG_1495](https://user-images.githubusercontent.com/6998954/120409556-2e1d3280-c31f-11eb-92e0-261105fa8f71.png)

Reply:
Okay